sdk/playwrighttesting/microsoft-playwright-testing/src/core/global/playwright-service-global-setup.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { dirname } from "path";
+import { FullConfig } from "../../common/types";
+import playwrightServiceEntra from "../playwrightServiceEntra";
+import { loadCustomerGlobalFunction } from "../../common/executor";
+import customerConfig from "../../common/customerConfig";
+
+const playwrightServiceGlobalSetupWrapper = async (config: FullConfig): Promise<any> => {
+  const rootDir = config.configFile ? dirname(config.configFile!) : process.cwd();
+  const customerGlobalSetupFunc = await loadCustomerGlobalFunction(
+    rootDir,
+    customerConfig.globalSetup,
+  );
+
+  await playwrightServiceEntra.globalSetup();
+  if (customerGlobalSetupFunc) {
+    // eslint-disable-next-line no-return-await
```

Comment:
I notice a lot of places where we disable no-return-await. Instead of disabling the linter rule, consider returning the promise directly if you're not planning to do anything with the result or handle exceptions.

The following are equivalent:
```ts
async function foo(): Promise<string> {
  return await someAsyncFunction();
}

function foo(): Promise<string> {
  return someAsyncFunction();
}
```

Where you must await is if you plan to do something with the result:

```ts
async function foo(): Promise<string> {
  const result = await someAsyncFunction();
  return result.toLowerCase(); // or whatever
}
```

or if you plan to handle exceptions / do post-processing:

```ts
async function foo(): Promise<string> {
  try {
    return await someAsyncFunction();
  } catch (err) {
    console.log(err);
  }
}

async function bar(): Promise<string> {
  try {
    return await someAsyncFunction();
  } finally {
    console.log("cleaning up");
  }
}
```

In those examples you'll also notice that the linter is smart enough not to complain about `no-return-await` as it is valid usage of `return await`